### PR TITLE
Update team section + formatting

### PR DESCRIPTION
**Pull request by ahmedhamodi**
Todo: update image for Anthony

Currently (as of May 24th) looks like:
image

### CODE REVIEW

File: src/data/data.json [hunk 1]
```diff
@@ -58,16 +58,23 @@
...
             "name": "Greg Maxin",
-            "job": "Co-founder"
+            "job": "Co-Founder & Director"
+        }
+    ],
+    "Team": [
+        {
+            "img": "img/team/stock_photo.jpg",
+            "name": "Anthony Robertson",
+            "job": "Business Development Executive"
```
Comment (ahmedhamodi) [inline src/data/data.json:77]: Thoughts on a better job title @GodGreg?

**Review by derekmaxin — COMMENTED** [review]
Reviewed and looks good to me.

**Review by GodGreg — APPROVED** [review]
Looks good!

**Comment by GodGreg** [comment]
> Reviewed and looks good to me.

Hey Derek. You only commented on the PR, could you approve it as well?

**Review by derekmaxin — APPROVED** [review]

**Review by derekmaxin — APPROVED** [review]

**Pull request merged**